ts/features/messages/utils/messages.ts

```diff
@@ -75,7 +75,7 @@ export const handleCtaAction = (
     const convertedLink = getInternalRoute(cta.action);
     // the service ID is specifically required for MyPortal webview usage,
```

Comment:
Abbattiamo MyPortal!!!!

Reply:
it shouldn't. we'll plan to remove it in a specific PR